.github/build-service-for-linux/build.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+cargo build --package clash-verge-self-service --release --target $INPUT_TARGET --features $INPUT_FEATURES
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)